init-kafka/src/main/java/io/strimzi/init/kafka/Main.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2017-2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.init.kafka;
+
+import io.fabric8.kubernetes.client.DefaultKubernetesClient;
+import io.fabric8.kubernetes.client.KubernetesClient;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.Map;
+
+public class Main {
```

Comment:
We probably should have unit tests for this? Or why do we think they are not needed?